ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -118,25 +119,27 @@ open class BottomCommandingController: UIViewController {
     public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
         super.traitCollectionDidChange(previousTraitCollection)
 
-        guard previousTraitCollection?.horizontalSizeClass != traitCollection.horizontalSizeClass else {
-            return
+        if previousTraitCollection?.horizontalSizeClass != traitCollection.horizontalSizeClass {
+            // On a horizontal size class change the top level sheet / bar surfaces get recreated,
+            // but the item views, containers and bindings persist and are rearranged during the individual setup functions.
+            if let bottomSheetController = bottomSheetController {
+                bottomSheetController.willMove(toParent: nil)
+                bottomSheetController.removeFromParent()
+                bottomSheetController.view.removeFromSuperview()
+            }
+            bottomSheetController = nil
+            bottomBarView?.removeFromSuperview()
+            bottomBarView = nil
+
+            if traitCollection.horizontalSizeClass == .regular {
+                setupBottomBarLayout()
+            } else {
+                setupBottomSheetLayout()
+            }
         }
 
-        // On a horizontal size class change the top level sheet / bar surfaces get recreated,
-        // but the item views, containers and bindings persist and are rearranged during the individual setup functions.
-        if let bottomSheetController = bottomSheetController {
-            bottomSheetController.willMove(toParent: nil)
-            bottomSheetController.removeFromParent()
-            bottomSheetController.view.removeFromSuperview()
-        }
-        bottomSheetController = nil
-        bottomBarView?.removeFromSuperview()
-        bottomBarView = nil
-
-        if traitCollection.horizontalSizeClass == .regular {
-            setupBottomBarLayout()
-        } else {
-            setupBottomSheetLayout()
+        if previousTraitCollection?.preferredContentSizeCategory != traitCollection.preferredContentSizeCategory {
```

Comment:
do we also want to check if previousTraitCollection is non-nil? Or is it intended to go through this code path if it's nil (is that the initial setup scenario?)

Reply:
From my testing the nil case doesn't seem to occur. Have you seen that happen? I'm not sure if we should be ignoring a potentially real trait change, even if previousTraitCollection is nil in some weird case after loadView.